queries.sql

```diff
@@ -108,18 +108,45 @@ FROM animals
 WHERE full_name = 'Jennifer Orwell'
     AND name LIKE '%mon';
 
-    SELECT name AS name_of_animal,
+SELECT name AS name_of_animal,
     full_name AS owner_full_name
 FROM animals
     JOIN owners ON animals.owner_id = owners.id
 WHERE full_name = 'Dean Winchester'
     AND escape_attempts = 0;
 
-    SELECT full_name AS owner_full_name,
+SELECT full_name AS owner_full_name,
     COUNT(owner_id) AS biggest_number_of_animals_owned
 FROM animals
     JOIN owners ON animals.owner_id = owners.id
 GROUP BY full_name
 ORDER BY biggest_number_of_animals_owned DESC
 LIMIT 1;
 
+-----Day4------ add "join table" for visits-------
+SELECT a.name last_animal_seen ,date_of_visit FROM animals a JOIN visits v ON a.id = v.animals_id JOIN vets vt ON vt.id = v.vets_id 
+WHERE vt.name = 'William Tatcher' ORDER BY date_of_visit DESC LIMIT 1; 
+
+SELECT COUNT(DISTINCT(v.animals_id)) total_seen FROM animals a JOIN visits v ON a.id = v.animals_id JOIN vets vt ON vt.id = v.vets_id 
+WHERE vt.name = 'Stephanie Mendez' GROUP BY vt.name;
+
+SELECT v.name,sp.name specialties FROM vets v LEFT JOIN specializations s ON v.id = s.vets_id LEFT JOIN species sp ON sp.id = s.species_id;
+
+SELECT a.name animal_visited ,date_of_visit FROM animals a JOIN visits v ON a.id = v.animals_id JOIN vets vt ON vt.id = v.vets_id 
+WHERE vt.name = 'Stephanie Mendez' AND date_of_visit BETWEEN '2020-04-01' AND '2020-08-30';
+
+SELECT a.name first_visited_animal,date_of_visit FROM animals a JOIN visits v ON a.id = v.animals_id JOIN vets vt ON vt.id = v.vets_id 
+WHERE vt.name = 'Maisy Smith' ORDER BY date_of_visit  LIMIT 1; 
```

Comment:
- [ ] Maisy smiths first visit is Boarmon, kindly fix the query, please.
![image](https://user-images.githubusercontent.com/94551484/228983601-9304583e-0cca-46cc-9a07-8c11c23398a5.png)